src/Search/Gearman/Command/QueueCommand.php

```diff
@@ -47,6 +48,9 @@ public function __construct(
         $this->logger = $logger;
         $this->monitoring = $monitoring;
         $this->limit = $limit;
+        // Signals.
+        Signals::register();
```

Comment:
if `Signals` is an object we can put the `->register()` call in the `bin/console` file like for https://github.com/elifesciences/search/blob/develop/bin/console#L30
which would make this class less dependent on it, and more testable as the constructor does not perform any special work when used inside a test case